src/commands/init.rs

```diff
@@ -80,12 +80,19 @@ pub(crate) fn save_config(
     let key = Key::new();
 
     let pass = password.map_or_else(
-        || match prompt_password("enter password for new key: ") {
-            Ok(it) => it,
-            Err(err) => {
-                status_err!("{}", err);
-                RUSTIC_APP.shutdown(Shutdown::Crash);
-            }
+        || -> String {
+            let password_result = Password::new().with_prompt("enter password for new key")
+                .with_confirmation("confirm password", "passwords do not match")
+                .interact();
+
+            let password = match password_result {
+                Ok(it) => it,
+                Err(err) => {
+                    status_err!("{}", err);
+                    RUSTIC_APP.shutdown(Shutdown::Crash);
+                }
+            };
+            password
```

Comment:
No need for new variables which are immediately used in the next command. You can use `match Password::new() ... { ... }` here.